index.html

```diff
@@ -287,6 +287,24 @@ <h2>About Myself</h2>
                 </ul>
             </div>
         </section>
+        <section class="contact">
+            <article class="contact_footer">
+                <h2>Contact me</h2>
+                <p>If you have an application you are interested
+                    in developing, a feature that you need built or a project
+                    that needs coding. <br>I'd
+                    love to help with it!</p>
+                <form class="form-start" action="https://formspree.io/f/xlekpydn"
+                method="POST">
+                    <input type="name" type="text" name="name" class="name" placeholder="write your name"
+                        maxlength="30" required>
+                    <input type="email" type="email" name="email" class="email" placeholder="write your email" required>
+                    <textarea type="message" type="text" name="message" class="message" placeholder="write your message"
+                        maxlength="500" required></textarea>
+                    <button class="start">Get in touch</button>
+                </form>
+            </article>
+        </section>
```

Comment:
- [ ] Great effort working on the contact section of the portfolio project 👍 I would like to highlight some changes that requires some attention.

1. I notice some extra grey color space at the left hand side of your contact section design, please kindly fix it and be sure it looks like what we have in the Figma design.
2. I notice you have duplicate the `type` attribute in your input as well as the text area, kindly be sure you are having only one type attribute as having them duplicated will make your code looks unprofessional.
3. Kindly give your input form some `border-radius` CSS property in order to look like what we have in the Figma design. As this will show you ability to replicate any given design and as well show your professionalism
4. Great job adding the button styling here but please make sure you stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=PBimssV7iaLZjIVt-0) as this ensures your future clients that you can implement a given design down to the last pixel 🔎.
Here's the design as a reminder: 😉


<img src="https://user-images.githubusercontent.com/104968236/230132266-c56d1c72-e3c3-4f3a-b8ce-071c1b929d67.png">

   - Also make sure to add the `:hover` and `:active(pressed)` states of the button as per the given design and also the border-radius should be implemented👆

| Your Design 👇 | Figma Design 👇 |
| ------------- | ------------- |
|  ![prad](https://user-images.githubusercontent.com/78634841/230162266-f056c665-dd29-4d7f-86d4-4eba19e735d6.png) | ![temp](https://user-images.githubusercontent.com/78634841/230162340-bbaf6920-5e5c-47df-a546-46f1a9b6414f.png) |